tests/test_tags/test_create_tags.py

```diff
@@ -57,3 +58,32 @@ def test_update_non_existent_tag(self):
             update_non_existent_tag_mutation,
             update_non_existent_tag_response
         )
+
+    def test_database_connection_error(self):
+        """
+        test a user friendly message is returned to a user when database
+        cannot be reached
+        """
+        BaseTestCase().tearDown()
+        CommonTestCases.admin_token_assert_in(
+            self,
+            create_tag_query,
+            "The database cannot be reached"
+            )
+        CommonTestCases.user_token_assert_in(
+            self,
+            update_tag_mutation,
+            "The database cannot be reached"
+        )
+
+    def test_create_tag_without_tags_model(self):
+        """
+        Test if a user can create a tag when there is a database error
```

Comment:
Should be "Test that a user cannot create a tag when there is a database error"

Reply:
Rectified @Bonifase